app/forms/support_interface/application_forms/edit_reference_form.rb

```diff
@@ -0,0 +1,46 @@
+module SupportInterface
+  module ApplicationForms
+    class EditReferenceForm
+      include ActiveModel::Model
+      include ValidationUtils
+
+      attr_accessor :name
+      attr_accessor :email_address
+      attr_accessor :relationship
+      attr_accessor :feedback
+      attr_accessor :audit_comment
+
+      attr_reader :reference
+      attr_reader :application_form
+
+      validates :name, presence: true, length: { minimum: 2, maximum: 200 }
+      validates :email_address, presence: true, email_address: true, length: { maximum: 100 }
+      validates :relationship, presence: true, word_count: { maximum: 50 }
+      validates :feedback, presence: true, word_count: { maximum: 500 }
```

Comment:
Is this always correct? We might want to edit a reference before feedback has been received

Reply:
Good point. I've also spoken to @paulrobertlloyd who agrees that we should split this out in to two separate forms. I've just pushed the changes